src/scss/skin-modern/components/_subtitleoverlay.scss

```diff
@@ -6,14 +6,22 @@
 
   @include hidden;
 
-  bottom: 2em;
+  align-items: flex-end;
+  display: flex;
   font-size: 1.2em;
-  left: 3em;
+  height: 86%;
+  justify-content: center;
+  margin: auto 10%;
   pointer-events: none;
   position: absolute;
-  right: 3em;
-  text-align: center;
```

Comment:
The removed text-alignment leads to wrapped text not being centered any more
![image](https://user-images.githubusercontent.com/189372/28173883-1be1405e-67f0-11e7-9590-75c23ff885f7.png)